executor/runner/runner.go

```diff
@@ -72,16 +73,17 @@ func New(ctx context.Context, m metrics.Reporter, logUploaders *uploader.Uploade
 	dockerRuntime := func(ctx context.Context, cfg config.Config) (runtimeTypes.Runtime, error) {
 		return docker.NewDockerRuntime(ctx, m, dockerCfg, cfg)
 	}
-	return WithRuntime(ctx, m, dockerRuntime, logUploaders, cfg)
+	return WithRuntime(ctx, m, dockerRuntime, logUploaders, cfg, &dockerCfg)
```

Comment:
`runner.go` is meant to be container runtime agnostic. `docker.Config` is specific to the Docker runtime. If there's a way we can avoid picking this up, that'd be ideal.

Reply:
docker.Config is already a parameter passed into the runner.New() function, so the Runner already knows about it.  I only pass that along into the runner.WithRuntime.  

Do you still want me to remove docker.Config from WithRuntime?  I can do it, but it won't address your comment about runner.go being agnostic of docker.Config.  Please let me know.

Reply:
New uses  Docker.Config because it's the default runtime. It gets passed through to the Docker runtime, but the actual runner doesn't use it at all. We could remove it, and hoist that to a higher layer if we wanted to, but for now, please don't add additional dependencies.

Reply:
I agree with your original comment.  I don't really need docker.Config.  I will remove it.